internal/token_test.go

```diff
@@ -270,6 +270,11 @@ func TestExpressions(t *testing.T) {
 			"{`${`${`${foo}`}`}`}",
 			[]TokenType{StartExpressionToken, TextToken, TextToken, TextToken, TextToken, TextToken, EndExpressionToken},
 		},
+		{
+			"element with multiple expressions",
+			"<div>Hello {first} {last}</div>",
```

Comment:
This was failing before. Basically, when entering expression context, we would enter “text” mode but fail to exit.